spec/beaker/cli_spec.rb

```diff
@@ -1,5 +1,15 @@
 require 'spec_helper'
 
+def load_yaml_file(path)
+  permitted = [Beaker::Options::OptionsHash, Symbol, RSpec::Mocks::Double, Time]
```

Comment:
The double is because the logger is being serialized for some reason. Looks like a bug to me, but I consider that out of scope.